checks/write.md

```diff
@@ -19,65 +19,63 @@ criteria and then create a new GitHub Issue to discuss with the team:
     of a given scorecard finding for that repository.
 -   Any criteria in the scorecard must be actionable. It should be possible,
     with help, for any project to "check all the boxes".
--   Any solution to compile a scorecard should be usable by the greater open
-    source community to monitor upstream security.
+-   Any solution to compile a scorecard should be usable by the greater
+    open-source community to monitor upstream security.
```

Comment:
Open style question here, whether to hyphenate open source as a compound adjective before a noun. Popular usage is trending toward dropping the hyphen, and there's little chance of confusion (nobody will misread it as open source-community).

Reply:
Sure thing. I actually thought this was a mandatory hyphen, not a stylistic choice. Learn something new every day.